product_docs/docs/epas/15/epas_guide/03_database_administration/01_configuration_parameters/03_configuration_parameters_by_functionality/05_query_tuning_planner_method_configuration.mdx

```diff
@@ -29,4 +29,4 @@ The ability to early-prune depends upon the nature of the query in the `WHERE` c
 Early-pruning is not used for more complex queries such as `WHERE column = expression` (e.g., `WHERE deptno = 10 + 5)`.
 
 !!! Note
-    This parameter is deprecated from version 15 onwards.
\ No newline at end of file
+    This parameter is deprecated from version 15 onwards, use `enable_partition_pruning` instead.
```

Comment:
use ; instead of ,

Reply:
I think this will change the meaning. I am trying to say here is the parameter `edb_enable_pruning` is deprecated and use `enable_partition_pruning` instead.